extension/src/context.ts

```diff
@@ -61,31 +59,16 @@ export class Context extends Disposable {
       return
     }
 
-    this.setContextFromRepositories(
+    void setContextValue(
       ContextKey.EXPERIMENT_RUNNING,
-      repositories,
-      (experiments: Experiments) => experiments.hasRunningExperiment()
-    )
-
-    const hasRunningQueuedExperiment = repositories.some(experiments =>
-      experiments.hasRunningQueuedExperiment()
+      repositories.some(experiments => experiments.hasRunningExperiment())
     )
 
     void setContextValue(
       ContextKey.EXPERIMENT_STOPPABLE,
-      hasRunningQueuedExperiment ||
-        (await this.experiments.hasDvcLiveOnlyExperimentRunning())
-    )
-  }
-
-  private setContextFromRepositories(
```

Comment:
[F] Realised in the previous PR that once we use `.some` we don't really need this function.